--- components/softdevice/common/nrf_sdh_ble.c	2019-02-14 17:24:58.000000000 +0100
+++ components/softdevice/common/nrf_sdh_ble.c	2019-07-03 13:40:29.903324300 +0200
@@ -110,7 +110,7 @@
         return ret_code;
     }
 
-#if defined (S112) || defined(S312)
+#if defined (S112) || defined(S312) || defined(S113)
     STATIC_ASSERT(NRF_SDH_BLE_CENTRAL_LINK_COUNT == 0, "When using s112, NRF_SDH_BLE_CENTRAL_LINK_COUNT must be 0.");
 #endif
 
@@ -137,7 +137,7 @@
     // Configure the connection roles.
     memset(&ble_cfg, 0, sizeof(ble_cfg));
     ble_cfg.gap_cfg.role_count_cfg.periph_role_count  = NRF_SDH_BLE_PERIPHERAL_LINK_COUNT;
-#if !defined (S112) && !defined(S312)
+#if !defined (S112) && !defined(S312) && !defined(S113)
     ble_cfg.gap_cfg.role_count_cfg.central_role_count = NRF_SDH_BLE_CENTRAL_LINK_COUNT;
     ble_cfg.gap_cfg.role_count_cfg.central_sec_count  = MIN(NRF_SDH_BLE_CENTRAL_LINK_COUNT,
                                                             BLE_GAP_ROLE_COUNT_CENTRAL_SEC_DEFAULT);
