diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/boards/pca10056.h b/external/nRF5_SDK_14.2.0_17b948a/components/boards/pca10056.h
index 53640dd56..6f1ca228a 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/boards/pca10056.h
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/boards/pca10056.h
@@ -53,6 +53,8 @@ extern "C" {
 #define LED_2          NRF_GPIO_PIN_MAP(0,14)
 #define LED_3          NRF_GPIO_PIN_MAP(0,15)
 #define LED_4          NRF_GPIO_PIN_MAP(0,16)
+#define LED_START      LED_1
+#define LED_STOP       LED_4
 
 #define LEDS_ACTIVE_STATE 0
 
diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/device/compiler_abstraction.h b/external/nRF5_SDK_14.2.0_17b948a/components/device/compiler_abstraction.h
index 3da528ac9..c69ec3787 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/device/compiler_abstraction.h
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/device/compiler_abstraction.h
@@ -115,13 +115,7 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
         #define __PACKED           __attribute__((packed)) 
     #endif
 
-    #define GET_SP()                gcc_current_sp()
-
-    static inline unsigned int gcc_current_sp(void)
-    {
-        register unsigned sp __ASM("sp");
-        return sp;
-    }
+    #define GET_SP()                __builtin_frame_address(0)
 
 #elif defined   ( __TASKING__ )
 
diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/drivers_nrf/hal/nrf_gpio.h b/external/nRF5_SDK_14.2.0_17b948a/components/drivers_nrf/hal/nrf_gpio.h
index ccd408c0a..0826438b9 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/drivers_nrf/hal/nrf_gpio.h
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/drivers_nrf/hal/nrf_gpio.h
@@ -42,7 +42,8 @@
 
 #include "nrf.h"
 #include "nrf_peripherals.h"
-#include "nrf_assert.h"
+/* #include "nrf_assert.h" */
+#define ASSERT(...)
 #include <stdbool.h>
 #include <stdlib.h>
 
diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf51.s b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf51.s
index 668f3cd8c..0f9fd7507 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf51.s
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf51.s
@@ -16,12 +16,12 @@
 ; 
 ; NOTICE: This file has been modified by Nordic Semiconductor ASA.
 
-                IF :DEF: __STARTUP_CONFIG
+#ifdef  __STARTUP_CONFIG
 #include "startup_config.h"
 #ifndef __STARTUP_CONFIG_STACK_ALIGNEMENT
 #define __STARTUP_CONFIG_STACK_ALIGNEMENT 3
 #endif
-                ENDIF
+#endif
 
                 IF :DEF: __STARTUP_CONFIG
 Stack_Size      EQU __STARTUP_CONFIG_STACK_SIZE
diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52.s b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52.s
index ac0493b69..205109789 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52.s
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52.s
@@ -16,12 +16,12 @@
 ; 
 ; NOTICE: This file has been modified by Nordic Semiconductor ASA.
 
-                IF :DEF: __STARTUP_CONFIG
+#ifdef  __STARTUP_CONFIG
 #include "startup_config.h"
 #ifndef __STARTUP_CONFIG_STACK_ALIGNEMENT
 #define __STARTUP_CONFIG_STACK_ALIGNEMENT 3
 #endif
-                ENDIF
+#endif
 
                 IF :DEF: __STARTUP_CONFIG
 Stack_Size      EQU __STARTUP_CONFIG_STACK_SIZE
diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52810.s b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52810.s
index 6f9d902e6..6faeb40a2 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52810.s
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52810.s
@@ -16,12 +16,12 @@
 ; 
 ; NOTICE: This file has been modified by Nordic Semiconductor ASA.
 
-                IF :DEF: __STARTUP_CONFIG
+#ifdef  __STARTUP_CONFIG
 #include "startup_config.h"
 #ifndef __STARTUP_CONFIG_STACK_ALIGNEMENT
 #define __STARTUP_CONFIG_STACK_ALIGNEMENT 3
 #endif
-                ENDIF
+#endif
 
                 IF :DEF: __STARTUP_CONFIG
 Stack_Size      EQU __STARTUP_CONFIG_STACK_SIZE
diff --git a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52840.s b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52840.s
index ff9d60ded..65a05a70b 100644
--- a/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52840.s
+++ b/external/nRF5_SDK_14.2.0_17b948a/components/toolchain/arm/arm_startup_nrf52840.s
@@ -16,12 +16,12 @@
 ; 
 ; NOTICE: This file has been modified by Nordic Semiconductor ASA.
 
-                IF :DEF: __STARTUP_CONFIG
+#ifdef  __STARTUP_CONFIG
 #include "startup_config.h"
 #ifndef __STARTUP_CONFIG_STACK_ALIGNEMENT
 #define __STARTUP_CONFIG_STACK_ALIGNEMENT 3
 #endif
-                ENDIF
+#endif
 
                 IF :DEF: __STARTUP_CONFIG
 Stack_Size      EQU __STARTUP_CONFIG_STACK_SIZE
